tests/ci/cdk/run-cdk.sh

```diff
@@ -178,7 +178,7 @@ function win_docker_img_build_status_check() {
 
 function build_linux_docker_images() {
   # Always destroy docker build stacks (which include EC2 instance) on EXIT.
-  trap destroy_docker_img_build_stack EXIT
+#  trap destroy_docker_img_build_stack EXIT
```

Comment:
same here, git keeps the history so just remove the code if not applicable anymore.